src/SpotifyApi.NetCore/ILibraryApi.cs

```diff
@@ -109,103 +95,110 @@ public interface ILibraryApi
-        /// <returns>A Task that, once successfully completed, returns a full <see cref="PagedAlbums"/> object.</returns>
+        /// <param name="accessToken">The bearer token which is gotten during the authentication/authorization process.</param>
+        /// <returns>A Task that, once successfully completed, returns a full <see cref="PagedTracks"/> object.</returns>
         /// <remarks>
-        /// https://developer.spotify.com/documentation/web-api/reference/library/get-users-saved-albums/
+        /// https://developer.spotify.com/documentation/web-api/reference/library/get-users-saved-tracks/
         /// </remarks>
-        Task<T> GetCurrentUsersSavedAlbums<T>(
+        Task<PagedTracks> GetUsersSavedTracks(
             int limit = 20,
             int offset = 0,
             string market = null,
             string accessToken = null
             );
         #endregion
 
-        #region GetUsersSavedShows
         /// <summary>
-        /// Get a list of shows saved in the current Spotify user’s library. Optional parameters can be used to limit the number of shows returned.
+        /// Get a list of the albums saved in the current Spotify user’s ‘Your Music’ library.
         /// </summary>
+        /// <param name="type">Required. Value is either album or track.</param>
         /// <param name="limit">Optional. The maximum number of objects to return. Default: 20. Minimum: 1. Maximum: 50.</param>
         /// <param name="offset">Optional. The index of the first object to return. Default: 0 (i.e., the first object). Use with limit to get the next set of objects.</param>
-        /// <returns>A Task that, once successfully completed, returns a full <see cref="PagedShows"/> object.</returns>
+        /// <param name="market">Optional. An ISO 3166-1 alpha-2 country code or the string from_token or the string from_token <see cref="SpotifyCountryCodes" />. Provide this parameter if you want to apply Track Relinking.</param>
+        /// <param name="accessToken">The bearer token which is gotten during the authentication/authorization process.</param>
+        /// <returns>A Task that, once successfully completed, returns a full <see cref="PagedAlbums"/> or <see cref="PagedTracks"/> object depending on the param type value.</returns>
         /// <remarks>
-        /// https://developer.spotify.com/documentation/web-api/reference/library/get-users-saved-shows/
+        /// https://developer.spotify.com/documentation/web-api/reference/library/get-users-saved-albums/
+        /// https://developer.spotify.com/documentation/web-api/reference/library/get-users-saved-tracks/
         /// </remarks>
-        Task<PagedShows> GetUsersSavedShows(
+        Task<T> GetCurrentUsersSavedAlbumsOrTracks<T>(
+            string type,
             int limit = 20,
             int offset = 0,
+            string market = null,
             string accessToken = null
             );
```

Comment:
`GetCurrentUsersSavedAlbumsOrTracks` should be an internal method and should not be on the Interface.